src/Analyzers/Core/Analyzers/RemoveUnnecessaryImports/AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer.cs

```diff
@@ -28,6 +28,11 @@ internal abstract class AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer
         // NOTE: This is a trigger diagnostic, which doesn't show up in the ruleset editor and hence doesn't need a conventional IDE Diagnostic ID string.
         internal const string DiagnosticFixableId = "RemoveUnnecessaryImportsFixable";
 
+        // NOTE: This is a special helper diagnostic ID which is reported when the remove unnecesssary diagnostic ID (IDE0005) is
+        // ecalated to a warning or an error, but 'GenerateDocumentationFile' is false, which leads to IDE0005 not being reported
+        // on command line builds. See https://github.com/dotnet/roslyn/issues/41640 for more details.
+        internal const string EnableGenerateDocumentationFileId = "EnableGenerateDocumentationFile";
```

Comment:
I am in two minds whether this diagnostic should get an `IDExxxx` ID or a special ID like this one is just fine.